### PR TITLE
Add ecobee humidifier

### PR DESCRIPTION
**Pull request by treylok**
## Proposed change

Document ability to set the humidifier mode of an ecobee thermostat with a humidifier attached.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45003
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This PR needs to be targeting the `next` branch, as it relies on a code change in the Home Assistant code base (and thus a next release).

Please change the target of this PR to the `next` branch and git rebase the PR if needed.

Thanks 👍

**Comment by frenck** [comment]
There are unrelated commits in this PR, you still need to rebase it onto the `next` branch @treylok 👍

**Comment by treylok** [comment]
Sorry, not sure what I did.  Hopefully this is correct.

File: source/_integrations/ecobee.markdown [hunk 1]
```diff
@@ -190,6 +193,15 @@ Enable/disable automatic daylight savings time.
 | `entity_id`            | yes      | ecobee thermostat on which to set daylight savings time mode |
 | `dst_enabled`          | no       | true or false                                     |
 
+### Service `ecobee.set_humidifier_mode`
+
+Set the humidifier mode.
+
+| Service data attribute | Optional | Description                                       |
+| ---------------------- | -------- | ------------------------------------------------- |
+| `entity_id`            | no       | ecobee thermostat on which to set the humidifier mode |
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: Entity ID is optional, as this service is registered with `async_register_entity_service`.

```suggestion
| `entity_id`            | yes      | ecobee thermostat on which to set the humidifier mode |
```

File: source/_integrations/ecobee.markdown [hunk 2]
```diff
@@ -124,6 +124,8 @@ The ecobee climate entity has some extra attributes to represent the state of th
 | `climate_mode` | This is the climate mode that is active, or would be active if no override is active.
 | `equipment_running` | This is a comma-separated list of equipment that is currently running.
 | `fan_min_on_time` | The minimum amount of time (in minutes) that the fan will run per hour. This is determined by the minimum fan runtime setting which can be changed in the ecobee app or on the thermostat itself.
+| `humidifier_mode` | This is only shown if a humidifier is controlled by ecobee and displays the current humidifier mode.
+| `humidifier_modes` | This is only shown if a humidifier is controlled by ecobee and displays the valid humidifier modes: `auto` / `off` / `manual`
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]:
```suggestion
| `humidifier_modes` | This is only shown if a humidifier is controlled by ecobee and displays the valid humidifier modes: `auto` / `off` / `manual`.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Yeah, that did the trick @treylok 👍 

2 small suggestions and the docs are ready to go 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/ecobee.markdown [hunk 3]
```diff
@@ -124,6 +124,8 @@ The ecobee climate entity has some extra attributes to represent the state of th
 | `climate_mode` | This is the climate mode that is active, or would be active if no override is active.
 | `equipment_running` | This is a comma-separated list of equipment that is currently running.
 | `fan_min_on_time` | The minimum amount of time (in minutes) that the fan will run per hour. This is determined by the minimum fan runtime setting which can be changed in the ecobee app or on the thermostat itself.
+| `humidifier_mode` | This is only shown if a humidifier is controlled by ecobee and displays the current humidifier mode.
+| `humidifier_modes` | This is only shown if a humidifier is controlled by ecobee and displays the valid humidifier modes: `auto` / `off` / `manual`.
```
Comment (MartinHjelmare) [inline source/_integrations/ecobee.markdown]: We changed the parent PR to add a humidifier platform instead.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Comment by MartinHjelmare** [comment]
Let's just update the `ha_category` and `ha_platforms` front matter.

File: source/_integrations/ecobee.markdown [hunk 4]
```diff
@@ -140,6 +142,7 @@ Besides the standard services provided by the Home Assistant [Climate](/integrat
 - `ecobee.resume_program`
 - `ecobee.set_fan_min_on_time`
 - `ecobee.set_dst_mode`
+- `ecobee.set_humidifier_mode`
```
Comment (MartinHjelmare) [inline source/_integrations/ecobee.markdown]: We didn't add this custom service.

Comment (treylok) [reply source/_integrations/ecobee.markdown]: Sorry I missed that one, removing now.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Review by frenck — APPROVED** [review]

**Pull request merged**